autogen.sh

```diff
@@ -2,8 +2,13 @@
 set -e
 srcdir="$(dirname $0)"
 cd "$srcdir"
+
 if [ -z ${LIBTOOLIZE} ] && GLIBTOOLIZE="`which glibtoolize 2>/dev/null`"; then
   LIBTOOLIZE="${GLIBTOOLIZE}"
   export LIBTOOLIZE
 fi
+if [ -z ${LIBTOOLIZE} ]; then
```

Comment:
`-z` is not the right check here, you probably want `! -x` to check for executableness